CMakeLists.txt

```diff
@@ -305,6 +305,7 @@ if (IGL_WITH_IGLU OR IGL_WITH_SAMPLES)
 
   igl_set_folder(obj_basisu_cbind "third-party/ktx-software")
   igl_set_folder(objUtil "third-party/ktx-software")
+  add_subdirectory(third-party/deps/src/fmt "fmt")
   igl_set_folder(fmt "third-party")
```

Comment:
FMT is added implicitly by other libraries. This should be:
```
if(TARGET fmt)
  igl_set_folder(fmt "third-party")
endif()
```

Reply:
Okay, I have modified it.